dcs/helicopters.py

```diff
@@ -2194,6 +2194,345 @@ class Pylon8:
     task_default = task.CAS
 
 
+class OH58D(HelicopterType):
+    id = "OH58D"
+    flyable = True
```

Comment:
To ensure other people that don't own the module also export this property correctly, you should add an entry in pydcs_export.lua as well. I'll track the approximate line-number down in a second...

Reply:
Insert at line 280 if you want to keep it in alphabetical order 😁